email_templates/plaintext/shutoff_error.txt.j2

```diff
@@ -1,15 +1,11 @@
 Dear Cloud User {{username}},
```

Comment:
We don't actually need this template any more - could you delete this template (and html equivalent) and also remove its entry in `email_template_schema.yaml` file